src/Formatter/ArrayBasedFormatter.php

```diff
@@ -103,6 +104,9 @@ public function serializeDictionary(mixed $runningValue, Field $field, Dict $nex
         $add = [];
         /** @var CollectionItem $item */
         foreach ($next->items as $item) {
+            if ($item->value === SerdeError::Uninitialized) {
+                continue;
+            }
```

Comment:
Skipping all uninitialized values in serialization basically here in order not to litter all exporters.

Reply:
Why would you even have an uninitialized sentinel value in the formatter?  Just skip it entirely further up, no?

Reply:
🤷 it seems like some dev relict, removed, testing passings...